### ISSUE TITLE
Icons not shown until more than half visible

### ISSUE THREAD

**Issue by Srekel** [OurMachinery/themachinery-public#534]
![image](https://user-images.githubusercontent.com/1578924/113481918-87dabb00-949c-11eb-9e6a-163e451078a1.png)

Happens in both grid and list view for asset browser. Also in the Entity Tree view.

**Comment by frank-ourmachinery** [comment]
I only seem to have this issue with the Entity (E) icon, the other ones don't seem to have this problem.

**Comment by niklas-ourmachinery** [comment]
Investigated and this seems to be an issue with clipping in the UI shader. If I disable clipping in `fill_triangles()` (which is used to draw the entity symbol) by doing:

~~~
if (cr == CLIP_RESULT_DISCARD)
    return;
else
    clip = 0;
~~~

The E renders correctly.

**Comment by niklas-ourmachinery** [comment]
@frank-ourmachinery : Could this be related to https://github.com/OurMachinery/themachinery/commit/a83ee8aa0513b8d8a33f4a8ff67adc23e773c200?

**Comment by niklas-ourmachinery** [comment]
Fixed for 2021.3.a in https://github.com/OurMachinery/themachinery/commit/0f254e9a10dfab790acabd11b79c95d5b9a40ffe.
